scripts/common.js

```diff
@@ -150,13 +150,14 @@ async function openExtensionTab(url) {
 
 async function openTab(tab, windowId, automatic = false) {
 	var windows = await getAllWindows();
+	var cookieStoreId = tab.cookieStoreId ? { cookieStoreId: tab.cookieStoreId } : {};
```

Comment:
Passing this property to `chrome.tabs.create` or `chrome.windows.create` causes an exception to be raised.

I thought about using `getBrowser() === 'firefox'` to conditionally pass this poperty, but I couldn't get the `getBrowser` function to behave (it returned `'chrome'` even in Firefox) and I didn't want to mix other fixes with this PR.